### PR TITLE
chore: Migrate event status VISITED to ACTIVE [TECH-608]

### PR DESCRIPTION
**Pull request by abyot**

### CODE REVIEW

File: dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/events/importer/shared/preprocess/EventStatusPreProcessorTest.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ * ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package org.hisp.dhis.dxf2.events.importer.shared.preprocess;
+
```
Comment (enricocolasante) [inline dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/events/importer/shared/preprocess/EventStatusPreProcessorTest.java:29]: Remove copyright after package

File: dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/events/importer/shared/preprocess/EventStatusPreProcessorTest.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    private EventStatusPreProcessor preProcessor;
+
+    @Before
+    public void setUp()
+    {
+        preProcessor = new EventStatusPreProcessor();
+    }
+
+    @Test
+    public void t1()
```
Comment (enricocolasante) [inline dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/events/importer/shared/preprocess/EventStatusPreProcessorTest.java:85]: Put a meaningful name to the test

File: dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/preprocess/EventStatusPreProcessorTest.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ * ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package org.hisp.dhis.tracker.preprocess;
+
+/*
```
Comment (enricocolasante) [inline dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/preprocess/EventStatusPreProcessorTest.java:30]: Same as before

**Review by enricocolasante — APPROVED** [review]
Small fixes to be adressed

**Review by ameenhere — APPROVED** [review]
Approved. Out of curiosity, I am sure there is a reason but why do we need the same file (EventStatusPreProcessor) in 2 different places?

File: dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/events/importer/shared/preprocess/EventStatusPreProcessorTest.java (hunk 1 above)
Comment (abyot) [reply dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/events/importer/shared/preprocess/EventStatusPreProcessorTest.java:29]: seems my code style auto format is messing up

File: dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/events/importer/shared/preprocess/EventStatusPreProcessorTest.java (hunk 2 above)
Comment (abyot) [reply dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/events/importer/shared/preprocess/EventStatusPreProcessorTest.java]: thanks @enricocolasante, not sure how that name endedup... have put a proper name.

**Comment by abyot** [comment]
> Approved. Out of curiosity, I am sure there is a reason but why do we need the same file (EventStatusPreProcessor) in 2 different places?

that is what came to my mind ... seems the old and the new services are using the same pre-processors

**Pull request merged**